Formula/ord.rb

```diff
@@ -0,0 +1,24 @@
+class Ord < Formula
+  desc "👁‍🗨 Rare and exotic sats"
```

Comment:
Could we choose a better description? I have no idea what this means.

Reply:
Updated to a more intuitive desc.